src/Map/Turn.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ChessDemo.Entities;
+
+namespace ChessDemo.Map
+{
+    class Turn
+    {
+        /// List of Teams to cycle through
+        private List<Team> teamList;
+
+        /// Dictionary of teams to entities
+        private Dictionary<Entity, Team> teamDict;
+
+        /// Team which is currently active
+        public Team currentTeam;
+
+        public Turn(Dictionary<Entity, Team> teams)
+        {
+            this.teamDict = teams;
+        }
+
+        public void initTeams() {
+
+            teamList = teamDict.Values.ToList();
+            currentTeam = teamList.First();
+        }
+
+        public Boolean myTurn(Entity currentEntity)
```

Comment:
Nit: Generally in C# methods start with a capital letter. The only thing with lowercase start is private variables